tests/conftest.py

```diff
@@ -67,3 +67,16 @@ def logout(self):
 @pytest.fixture
 def auth(client):
     return AuthActions(client)
+
+class Newsletter(object):
+    def __init__(self, client):
+        self._client = client
+
+    def signup(self, email='test'):
+        return self._client.post(
+            '/newsletter/signup',
+            data={'email': email}
+        )
+
+def newsletter(client):
+    return Newsletter(client)
```

Comment:
Is this a fixture that helps out with some other newsletter tests? If so, can you move it into its own file for newsletter tests `test_newsletter.py` or into `test_create_user_email.py`?

We DO have some fixtures for authentication stuff in here, but many different use cases might require authentication, whereas this setup would only affect closely-related use cases (signing up for the newsletter and maybe also sending a newsletter email if we do that through the webserver).

Reply:
tbh i'm not sure what this does, i was doing a lot of copy/pasting so I don't know if we need this or not. I kept it just in case and moved it to test_create_newsletter_subscriber.py (renamed to match table)